README.md

```diff
@@ -1,6 +1,10 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+For instructions, 
+--you need to install dotenv package.
```

Comment:
This is already in your package.json with your other package dependencies, so you don't need to specify it here.
It would be more useful to remind any devs who want to clone and use your repo to do _something else_ to install all the dependencies at once... 👀